cmd/wallet-web/src/layouts/WACI.vue

```diff
@@ -60,3 +84,13 @@ export default {
   },
 };
 </script>
+<style>
+.oval {
+  left: 50%;
+  transform: translateX(-50%);
+  border-radius: 50%;
+  filter: blur(50px);
```

Comment:
does blur function only accepts px ??

Reply:
No, we can also use `rem`. I'm not sure if it's need in this case though. Let me check with the design team

Reply:
They mentioned this is fine